dev-lang/erlang/erlang-21.0.5.ebuild

```diff
@@ -0,0 +1,220 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+WX_GTK_VER="3.0-gtk3"
+
+inherit autotools elisp-common java-pkg-opt-2 systemd versionator wxwidgets
+
+# NOTE: If you need symlinks for binaries please tell maintainers or
+# open up a bug to let it be created.
+
+UPSTREAM_V="$(get_version_component_range 1-2)"
+
+DESCRIPTION="Erlang programming language, runtime environment and libraries (OTP)"
+HOMEPAGE="http://www.erlang.org/"
+SRC_URI="https://github.com/erlang/otp/archive/OTP-${PV}.tar.gz -> ${P}.tar.gz
+	http://erlang.org/download/otp_doc_man_${UPSTREAM_V}.tar.gz -> ${PN}_doc_man_${UPSTREAM_V}.tar.gz
+	doc? ( http://erlang.org/download/otp_doc_html_${UPSTREAM_V}.tar.gz -> ${PN}_doc_html_${UPSTREAM_V}.tar.gz )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~x64-solaris"
+
+IUSE="doc emacs +hipe java +kpoll libressl odbc +pgo sctp ssl systemd tk +wxwidgets"
+
+RDEPEND="
+	sys-libs/ncurses:0
+	sys-libs/zlib
+	emacs? ( virtual/emacs )
+	java? ( virtual/jdk:* )
+	odbc? ( dev-db/unixODBC )
+	sctp? ( net-misc/lksctp-tools )
+	ssl? (
+		!libressl? ( >=dev-libs/openssl-0.9.7d:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	systemd? ( sys-apps/systemd )
+	tk? ( dev-lang/tk:0 )
+	wxwidgets? ( x11-libs/wxGTK:${WX_GTK_VER}[X,opengl] )
+"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+"
+
+S="${WORKDIR}/otp-OTP-${PV}"
+
+PATCHES=(
+		"${FILESDIR}/18.2.1-wx3.0.patch"
+		"${FILESDIR}/${PN}-20.3.2-dont-ignore-LDFLAGS.patch"
+		"${FILESDIR}/${PN}-add-epmd-pid-file-creation-for-openrc.patch"
+	)
+
+SITEFILE=50"${PN}"-gentoo.el
+
+# Taken from the upstream build script, build_otp
+erlang_create_lib_configure_in() {
+	local bootstrap_apps="erts lib/asn1 lib/compiler lib/hipelib/ic lib/kernel
+			 lib/parsetools lib/sasl lib/snmp lib/stdlib lib/syntax_tools"
+	local sdirs=
+	for lib_app in ${bootstrap_apps}; do
+		case "${lib_app}" in
+			lib/*)
+				if [[ -f "${lib_app}/configure.in" ]]; then
+					mv "${lib_app}/configure.in" "${lib_app}/configure.ac" || die
+					app=`echo "${lib_app}" | sed "s|lib/\(.*\)|\1|"`
+					sdirs="${sdirs}test ! -f ${app}/configure || AC_CONFIG_SUBDIRS(${app}/.)\n" || die
```

Comment:
Why do you need to do such a horrible hack? This looks really fragile, using autoconf macros on RHS of `||`.

Also, use `+=`.